### PR TITLE
feat: add age_group field in ticket holder model

### LINKED ISSUE #6742: Add ageGroup field in Attendee Model
Related Front-End Issue - [#3796](https://github.com/fossasia/open-event-frontend/issues/3796)

add `ageGroup` field in attndee model for the same.

**Additional context**

I am working on this issue.

### PR DESCRIPTION
**Pull request by snitin315**
Fixes #6742 

#### Changes proposed in this pull request:

added `age_group` field in attendee schema and corresponding model.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: app/models/custom_form.py [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "facebook": {"include": 0, "require": 0},
     "github": {"include": 1, "require": 0},
     "gender": {"include": 0, "require": 0},
+    "age_group": {"include": 0, "require": 0},
```
Comment (kushthedude) [inline app/models/custom_form.py]: Remove from here

**Comment by iamareebjamal** [comment]
This will accept any string as age group which was never the intention

**Comment by iamareebjamal** [comment]
Doesn't matter really. I can set `ooga booga` and server will still accept it. Frontend checks are as useless as Windows Defender against viruses

**Comment by snitin315** [comment]
> Doesn't matter really. I can set `ooga booga` and server will still accept it. Frontend checks are as useless as Windows Defender against viruses

Any suggetions whats should be done?

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4826893&bid=16040620)

File: app/api/helpers/static.py [hunk 2]
```diff
@@ -423,3 +423,11 @@
...
     'USD'
 ]
+
+AGE_GROUP_CHOICES = [
+    '19 or less',
+    '20 to 29',
+    '30 to 39',
+    '40 to 49',
+    '50 or above'
+]
```
Comment (niranjan94) [inline app/api/helpers/static.py]: Codacy Issue found: [Final newline missing](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4826893)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4826893&bid=16040620)

File: app/api/helpers/static.py (hunk 2 above)
Comment (niranjan94) [inline app/api/helpers/static.py]: Codacy Issue found: [Final newline missing](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4826893)

**Comment by iamareebjamal** [comment]
This should be dynamic but will do it for now

File: app/api/helpers/static.py [hunk 3]
```diff
@@ -423,3 +423,12 @@
...
+
```
Comment (iamareebjamal) [inline app/api/helpers/static.py]:
```suggestion
```

File: app/api/helpers/static.py (hunk 2 above)
Comment (iamareebjamal) [inline app/api/helpers/static.py]:
```suggestion
]

```

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4826893&bid=16040620)

File: app/api/helpers/static.py (hunk 2 above)
Comment (niranjan94) [inline app/api/helpers/static.py]: Codacy Issue found: [Final newline missing](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4826893)

**Pull request merged**